packages/node_modules/@node-red/runtime/lib/nodes/Node.js

```diff
@@ -224,7 +224,7 @@ Node.prototype._emitInput = function(arg) {
                     }
                 );
             } catch(err) {
-                node.error(err,msg);
+                node.error(err);
```

Comment:
`msg` might not be in scope, but `arg` is, and that's what should be passed here as you can is done on line 205.
```suggestion
                node.error(err,arg);
```

Reply:
Makes sense!